app/pdf_utils/fill_sections.py

```diff
@@ -66,4 +63,17 @@ def get_paragraph_text(section_element):
             # we will return the reference content if encountered
             return reference(refId)
 
+        # # TODO allow inline text formatting
```

Comment:
Hey @leothomas  
In this section, I'm looking to implement the last functionality that allows inline formatting of text (bold, italic, etc).
Do you know how text needs to be "wrapped" or returned in order for the formatting to be preserved and translated into the PDF? 
So if you uncomment lines 67 through the return statement, you'll see that the `command()` function is formatting the text according to the `TEXT_WRAPPERS`, but the format syntax `superscript...etc etc` shows up in the PDF.

Reply:
I'm also looking to include line break for sections of the `document_data` that are empty strings within paragraph text leaf format i.e. paragraph line break that the user input.